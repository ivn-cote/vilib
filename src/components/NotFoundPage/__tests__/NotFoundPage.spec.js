import React from 'react';
import renderer from 'react-test-renderer';
import NotFoundPage from '../index';

describe('<NotFoundPage />', () => {
  it('should have a proper markup', () => {
    const tree = renderer.create(
      <NotFoundPage />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
