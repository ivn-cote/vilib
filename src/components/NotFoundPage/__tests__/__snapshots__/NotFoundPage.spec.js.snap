// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should have a proper markup 1`] = `
<div>
  <h1>
    404 - Page Not Found
  </h1>
</div>
`;
